spec/features/register_names_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './gamechoice'
+
+SEED = 7654
+feature 'Register names' do
+  scenario 'name registered before playing' do
+    visit('/')
+    fill_in :player_1_name, with: 'Jake'
+    click_button 'Register'
+    expect(page).to have_content 'Jake the marketeer'
+  end
+end
+
+feature 'Playing a game of RPS' do
+  scenario 'Move choices are displayed as buttons' do
+  visit('/play')
+  expect(page).to have_button 'Rock'
+  expect(page).to have_button 'Paper'
+  expect(page).to have_button 'Scissors'
+  end
+end
```

Comment:
Good way of condensing down the tests, but I believe to make your tests easier to spot what went wrong, it is recommended to split these `expect(page)` into different tests.